Packs/MicrosoftGraphApplications/Integrations/MicrosoftGraphApplications/MicrosoftGraphApplications_test.py

```diff
@@ -137,3 +144,266 @@ def test_service_principal_list_command(mocker, requests_mock,
+    """
+    from MicrosoftGraphApplications import remove_service_principals_command
+    mock_token = {'access_token': 'test_token', 'expires_in': '86400'}
+    requests_mock.post('https://login.microsoftonline.com/organizations/oauth2/v2.0/token', json=mock_token)
+
+    with pytest.raises(DemistoException,
+                       match=re.escape("Either the (object's) `id` or the `application_id` arguments must be provided.")):
+        remove_service_principals_command(my_client, args={})
+
+
+GET_SERVICE_PRINCIPAL_RESPONSE = {'@odata.context': 'https://graph.microsoft.com/v1.0/$metadata#servicePrincipals/$entity',
+                                  'id': 'XXXX', 'deletedDateTime': None, 'accountEnabled': True, 'alternativeNames': [],
+                                  'appDisplayName': 'Test', 'appDescription': None,
+                                  'appId': 'XXXX', 'applicationTemplateId': None, 'appOwnerOrganizationId': 'XXXX',
+                                  'appRoleAssignmentRequired': False, 'createdDateTime': '', 'description': None,
+                                  'disabledByMicrosoftStatus': None, 'displayName': 'Test', 'homepage': None, 'loginUrl': None,
+                                  'logoutUrl': None, 'notes': None, 'notificationEmailAddresses': [],
+                                  'preferredSingleSignOnMode': None, 'preferredTokenSigningKeyThumbprint': None, 'replyUrls': [],
+                                  'servicePrincipalNames': ['XXXX'], 'servicePrincipalType': 'Application',
+                                  'signInAudience': 'AzureADMyOrg',
+                                  'tags': ['HideApp', 'WindowsAzureActiveDirectoryIntegratedApp'], 'tokenEncryptionKeyId': None,
+                                  'samlSingleSignOnSettings': None, 'addIns': [], 'appRoles': [],
+                                  'info': {'logoUrl': None, 'marketingUrl': None, 'privacyStatementUrl': None, 'supportUrl': None,
+                                           'termsOfServiceUrl': None}, 'keyCredentials': [], 'oauth2PermissionScopes': [],
+                                  'passwordCredentials': [], 'resourceSpecificApplicationPermissions': [],
+                                  'verifiedPublisher': {'displayName': None, 'verifiedPublisherId': None, 'addedDateTime': None}}
+
+
+def test_get_service_principal_command(requests_mock, my_client):
+    """
+        Given:
+            - Required arguments (id or app_id or both)
+        When:
+            - Executing msgraph-apps-service-principal-get command
+        Then:
+            - Ensure the CommandResults contains the expected result
+    """
+    from MicrosoftGraphApplications import get_service_principal_command
+
+    mock_token = {'access_token': 'test_token', 'expires_in': '86400'}
+    requests_mock.post('https://login.microsoftonline.com/organizations/oauth2/v2.0/token', json=mock_token)
```

Comment:
are these still needed? (same for all other cases added in this PR, where the fixture is used)

Reply:
Removed them when they weren't needed